File: Solution/WebSolution/Java/AssessmentAPI/src/main/java/com/transflower/tflassessment/services/AssessmentIntelligenceService.java
package com.transflower.tflassessment.services;

import java.util.List;

import com.transflower.tflassessment.entities.AnnualCandidateResult;

public interface AssessmentIntelligenceService {
    List <AnnualCandidateResult> getCandidateResults(int candidateId, int year);
}

File: Solution/dotnet/legacycode/AssessmentPortal/Monolithic/Monolithic_Controller/AssessmentServices/Implementations/AssessmentIntelligenceService.cs
using Transflower.TFLAssessment.Entities;
using Transflower.TFLAssessment.Repositories.Interfaces;
using Transflower.TFLAssessment.Services.Interfaces;

namespace Transflower.TFLAssessment.Services{
public class AssessmentIntelligenceService:IAssessmentIntelligenceService
{
    private readonly IAssessmentIntelligenceRepository _repo;

    public AssessmentIntelligenceService(IAssessmentIntelligenceRepository repository){
        _repo=repository;
    }

    public async Task<List<AnnualCandidateResult>> GetCandidateResults(int candidateId, int year){
        return await _repo.GetCandidateResults(candidateId, year);
    }

    
}
}

